tt1-blocks/style.css

```diff
@@ -46,7 +46,8 @@ body {
 }
 
 .wp-site-blocks > *:not(.wp-block-post-content),
-.wp-site-blocks .wp-block-post-content > * {
+.wp-site-blocks .wp-block-post-content > *,
+.wp-block-query-loop li > *:not(.entry-content):not(.alignwide):not(.alignfull) {
```

Comment:
did this creep in by mistake?

Reply:
No, I just added that. Previously, all content of the query block was showing up as standard width, even when it was wide/full. 😩 I noticed this when I moved the default query into its own template part, and had to do this to fix it.

Here's an example, where the Media & Text block is supposed to be wide width:

Before f4a0131|After I placed the query block in the full-width template part.|Final result, after I added this css.
---|---|---
<img width="1395" alt="Screen Shot 2020-12-18 at 9 12 03 AM" src="https://user-images.githubusercontent.com/1202812/102623913-3d157d00-4111-11eb-8676-788602437306.png">|<img width="1395" alt="Screen Shot 2020-12-18 at 9 11 24 AM" src="https://user-images.githubusercontent.com/1202812/102623948-4acb0280-4111-11eb-947e-592595252a4d.png">|<img width="1395" alt="Screen Shot 2020-12-18 at 9 11 09 AM" src="https://user-images.githubusercontent.com/1202812/102623962-50284d00-4111-11eb-99d4-31d63322585c.png">




(I _really_ can't wait until this alignments stuff "just works" 😞).